staramr/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# base image
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="staramr"
+LABEL software.version="0.5.1"
+LABEL description="staramr scans bacterial genome contigs for AMR genes and plasmids"
+LABEL website="https://github.com/phac-nml/staramr"
+LABEL license="https://github.com/phac-nml/staramr"
+
+# Maintainer
+MAINTAINER Rachael St. Jacques <rachael.stjacques@dgs.virginia.gov>
+
+# install dependencies using conda
```

Comment:
omit/change the comment about dependencies. probably left over from copy-pasting from another dockerfile....just a little misleading since conda isn't used here